presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/ExpressionVerifier.java

```diff
@@ -174,7 +175,10 @@ protected Boolean visitDoubleLiteral(DoubleLiteral actual, Expression expected)
     @Override
     protected Boolean visitBooleanLiteral(BooleanLiteral actual, Expression expected)
     {
-        return getValueFromLiteral(actual).equals(getValueFromLiteral(expected));
+        if (expected instanceof BooleanLiteral) {
+            return getValueFromLiteral(actual).equals(getValueFromLiteral(expected));
```

Comment:
it is in its own commit?  It's necessary because exist queries add a project true, and without this change, that gets evaluated incorrectly. (Basically this gets called in a loop on a list of the expected expressions, and the result is or-ed together.  Without this change, it would just throw an exception here because some of the expressions weren't booleans, even though there was a true expression later in the list).